bin/jpm

```diff
@@ -69,9 +70,15 @@ program
 program
   .command("xpi")
   .description("Bundle the addon into an .xpi file")
-  .action(cmd.prepare("xpi", program, function(manifest) {
+  .option("--dest-dir <output_dir>", "Where to write the generated xpi",
+          // turn into an absolute path
+          function(dir) {
+            return path.resolve(dir);
+          },
+          process.cwd())
```

Comment:
The problem with the test failing is here. The `process.cwd()` changes the default location for the xpi when `--dest-dir` is not specified.

Currently the test effectively invokes jpm with `--addon-dir` set to a path, but not setting the `--dest-dir`. Hence the output directory is now set to `process.cwd()` whereas before it would have been the same location as `--addon-dir`.

I'd suggest just dropping the `process.cwd()` argument from this completely, and then when `--dest-dir` isn't specified we'll get the same behavior as jpm currently has.